unable to make buildoss

go get -d github.com/cockroachdb/cockroach and after that run make buildoss

host is fedora 28
```
build/node-run.sh -C pkg/ui ./node_modules/.bin/webpack --progress -p --config webpack.protos.js --env.dist=oss
Hash: 4e66ad26dbfe6374864e
Version: webpack 3.8.1
Time: 148ms
            Asset       Size  Chunks             Chunk Names
protos.oss.dll.js  530 bytes       0  [emitted]  protos
   [0] dll protos 12 bytes {0} [built]

ERROR in dll protos
Module not found: Error: Can't resolve './src/js/protos' in '/root/.cache/go-path/src/github.com/cockroachdb/cockroach/pkg/ui'
 @ dll protos
make: *** [Makefile:1152: pkg/ui/dist/protos.oss.dll.js] Error 2
make: *** Deleting file 'pkg/ui/dist/protos.oss.dll.js'
make: *** [pkg/ui/dist/protos.oss.dll.js] Deleting file 'pkg/ui/protos.oss.manifest.json'
```

--- a/Makefile
+++ b/Makefile
@@ -1038,7 +1038,7 @@ UI_PROTOS_CCL := $(UI_JS_CCL) $(UI_TS_CCL)
 
 UI_JS_OSS := pkg/ui/src/js/protos.js
 UI_TS_OSS := pkg/ui/src/js/protos.d.ts
-UI_PROTOS_OSS := $(UI_JS) $(UI_TS)
+UI_PROTOS_OSS := $(UI_JS_OSS) $(UI_TS_OSS)
 
 CPP_PROTOS := $(filter %/roachpb/metadata.proto %/roachpb/data.proto %/roachpb/internal.proto %/engine/enginepb/mvcc.proto %/engine/enginepb/mvcc3.proto %/engine/enginepb/file_registry.proto %/engine/enginepb/rocksdb.proto %/hlc/legacy_timestamp.proto %/hlc/timestamp.proto %/unresolved_addr.proto,$(GO_PROTOS))
 CPP_HEADERS := $(subst ./pkg,$(CPP_PROTO_ROOT),$(CPP_PROTOS:%.proto=%.pb.h))

